test/e2e/proza-message.test.js

```diff
@@ -9,4 +9,63 @@ describe('vl-proza-message', async () => {
         return vlProzaMessagePage.load();
     });
 
+    it('de tekst word aanpasbaar als ik op het potlood klik', async () => {
+        const message = await vlProzaMessagePage.getMessageFirstDemo();
+        await message.clickOnPencil();
+        await assert.eventually.isTrue(message.isEditable());
+        await message.exitEditMode();
+    });
+
+    it('als de gebruiker op de escape-toets drukt, worden de wijzigingen niet bewaard', async () => {
+        const message = await vlProzaMessagePage.getMessageFirstDemo();
+        await message.clickOnPencil()
+        await message.clear();
+        await message.type('decibel')
+        await message.exitEditMode();
```

Comment:
hier dan misschien exitEditModeWithoutSaving of zo van maken?

Reply:
aardappel aardappel